### PR TITLE
Adding documentation for Roku hub and remote

### PR DESCRIPTION
**Pull request by soberstadt**
**Description:**
Updating documentation to support Roku hub and remote.

As with the matching PR, this was mostly copied from @alandtse's #4713. Hopefully I have more free-time to be able to get these across the finish line :)

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#17548

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/media_player.roku.markdown [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
 layout: page
-title: "Roku"
+title: "Roku Media Player"
 description: "Instructions on how to integrate Roku into Home Assistant."
-date: 2016-05-16 20:0+0000
+date: 2018-10-17 20:0+0000
```
Comment (frenck) [inline source/_components/media_player.roku.markdown]: Please do not update these, the dates indicate the original creation.

**Review by frenck — CHANGES_REQUESTED** [review]
@soberstadt One small comment, could you take a look?

File: source/_components/media_player.roku.markdown (hunk 1 above)
Comment (soberstadt) [reply source/_components/media_player.roku.markdown]: Fixed! Thanks!

**Comment by soberstadt** [comment]
I've also updated to latest next.

File: source/_components/remote.roku.markdown [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+title: "Roku Remote"
+description: "Instructions on how to integrate Roku into Home Assistant."
+date: 2018-10-17 20:0+0000
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: roku.png
+ha_category: Remote
+ha_release: 0.81
```
Comment (MartinHjelmare) [inline source/_components/remote.roku.markdown]: 0.86

File: source/_components/remote.roku.markdown [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+- reverse
+- right
+- search
+- select
+- up
+- volume_down
+- volume_mute
+- volume_up
+
+A typical service call for press several buttons looks like this.
```
Comment (MartinHjelmare) [inline source/_components/remote.roku.markdown]: for pressing several buttons

File: source/_components/roku.markdown [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+description: "Instructions how to integrate Roku devices into Home Assistant."
+date: 2018-10-17 20:47
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: roku.png
+ha_category: Hub
+ha_iot_class: "Local Push"
+ha_release: 0.81
```
Comment (MartinHjelmare) [inline source/_components/roku.markdown]: 0.86

File: source/_components/roku.markdown [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+---
+
+The [Roku](http://www.roku.com/) platform allows integration of Roku [media players](/components/media_player.roku) and [remotes](/components/remote.roku/) which will be automatically discovered if you enable the [discovery component](/components/discovery/).
```
Comment (MartinHjelmare) [inline source/_components/roku.markdown]: The Roku component allows integration...

File: source/_components/roku.markdown [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+The `roku` platform can also be forced to load by adding the following lines to your `configuration.yaml`:
```
Comment (MartinHjelmare) [inline source/_components/roku.markdown]: It's a component.

File: source/_components/roku.markdown [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+title: "Roku"
+description: "Instructions how to integrate Roku devices into Home Assistant."
+date: 2018-10-17 20:47
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: roku.png
+ha_category: Hub
+ha_iot_class: "Local Push"
```
Comment (MartinHjelmare) [inline source/_components/roku.markdown]: Local Polling

**Comment by soberstadt** [comment]
@MartinHjelmare I have addressed all of your comments. Thanks for the review!

**Review by MartinHjelmare — APPROVED** [review]

**Comment by MartinHjelmare** [comment]
@frenck?

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
LGTM, thanks @soberstadt! 👍

**Pull request merged**